packages/commercetools/theme/pages/Checkout/Payment.vue

```diff
@@ -5,7 +5,7 @@
       title="Payment"
       class="sf-heading--left sf-heading--no-underline title"
     />
-    <SfAccordion first-open class="accordion smartphone-only">
+    <SfAccordion open="Shipping address" class="accordion smartphone-only">
```

Comment:
Translations needed, I guess. Line below too

Reply:
Please check other accordions too

Reply:
in this case line 8 doesn't need translations - it just indicates which title should be opened but you're right about a line below  :)

Reply:
But let's imagine:
SfAccordionItem.header = $t('Shipping address') which returns `Versandart`
and in `SfAccordion` you have `open = 'Shipping address'` instead of `open = $t('Shipping address')`.
Will it work?